src/games/strategy/engine/data/export/GameDataExporter.java

```diff
@@ -212,14 +212,19 @@ private void printEditableProperty(final IEditableProperty prop) {
 
   private void printConstantProperties(final Map<String, Object> conProperties) {
     for(String propName : conProperties.keySet()) {
-      if (propName.equals("notes")) { // TODO: unchecked reflection
-        // Special handling of notes property
-        printNotes((String) conProperties.get(propName));
-      } else if (propName.equals("EditMode") || propName.equals("GAME_UUID")
-          || propName.equals("games.strategy.engine.framework.ServerGame.GameHasBeenSaved")) {
-        // Don't print these options
-      } else {
-        printConstantProperty(propName, conProperties.get(propName));
+      switch (propName) {
```

Comment:
Not sure this is actually easier to read/maintain converting to a switch. Once you have logic in checks where multiple cases map to the same logic, its probably easier to read as if/else.